app/screens/navigation.ts

```diff
@@ -677,19 +677,24 @@ export function showModalOverCurrentContext(name: AvailableScreens, passProps =
 }
 
 export async function dismissModal(options?: Options & { componentId: AvailableScreens}) {
```

Comment:
Should these changes be done here? Keep in mind this function is being called through all the app, so not sure if every single instance of dismiss modal should check the is table to emit this event.

Regarding the keyboard dismiss... I have mixed feelings, because a part of me thinks it is nice to have it here, but not in every call to dismiss modal are we doing it, so I am worried about changes in the behavior.

Since this is supposed to fix a small bug, I would keep the changes in the component file, but 0/5.

Reply:
Valid point, I made changes to kept the function in same file.